### PR TITLE
Added lost changes

### PR DESCRIPTION
**Pull request by onionka**

### CODE REVIEW

File: proto/v1/admin_api.proto [hunk 1]
```diff
@@ -103,28 +103,78 @@ service StoreService {
...
-    rpc CreateStore (Store) returns (Store) {
+    rpc CreateStore (CreateStoreRequest) returns (Store) {
+        option (google.api.http) = {
+            post: "/merchants/{merchantId}/stores"
+            body: "*"
+        };
+    }
+
+    // Create store creates the store for the particular merchant and returns
+    // the store object back to the caller
```
Comment (radobar) [inline proto/v1/admin_api.proto]: This is update, not create.

**Review by rysavyvladan — CHANGES_REQUESTED** [review]
You need to regenerate files

File: proto/v1/admin_api.proto [hunk 2]
```diff
@@ -103,28 +103,78 @@ service StoreService {
...
 
     // Loads all stores from given merchant
-    rpc GetStores (Store) returns (StoreList) {
+    rpc GetStores (GetStoresRequest) returns (StoreList) {
         option (google.api.http) = {
             get: "/merchants/{merchantId}/stores"
         };
     }
+
+    // Loads single stores from given merchant
```
Comment (radobar) [inline proto/v1/admin_api.proto]:
```suggestion
    // Loads single store from a given merchant
```

File: proto/v1/admin_api.proto [hunk 3]
```diff
@@ -103,28 +103,78 @@ service StoreService {
...
+message GetStoreRequest {
     string storeId = 1;
-    string name = 2;
-    string location = 3;
+    string merchantId = 2;
+}
 
-    // [Required] reference to the merchant id
-    string merchantId = 10;
+// Request for created for create
```
Comment (radobar) [inline proto/v1/admin_api.proto]: This does not make sense.

File: proto/v1/admin_api.proto [hunk 4]
```diff
@@ -103,28 +103,78 @@ service StoreService {
...
+    string storeId = 1;
+    string merchantId = 2;
+
+    string name = 10;
+    string location = 11;
+    string address = 12;
+    repeated InventoryItem items = 13;
+}
+
+// Request for created for create
```
Comment (radobar) [inline proto/v1/admin_api.proto]: Neither this.

File: proto/v1/admin_api.proto [hunk 5]
```diff
@@ -134,22 +184,22 @@ message StoreList {
 // Inventory service is implementing storing concrete products (Item model) that are
 // in N:1 relationship with Product model.
 service InventoryService {
-    // Loads all stores from given merchant
-    rpc AddInventoryItem (CreateInventoryItemRequest) returns (InventoryItem) {
+    // Adds inventory item into the inventory of merchant.
```
Comment (radobar) [inline proto/v1/admin_api.proto]:
```suggestion
    // Adds an inventory item into the inventory of a merchant.
```

File: proto/v1/admin_api.proto [hunk 6]
```diff
@@ -134,22 +184,22 @@ message StoreList {
...
+    rpc AddInventoryItem (AddInventoryItemRequest) returns (InventoryItem) {
         option (google.api.http) = {
             post: "/merchants/{merchantId}/inventories/{inventoryId}/items"
             body: "*"
         };
     }
 
-    // Loads all stores from given merchant
+    // Loads all stores from given merchant.
```
Comment (radobar) [inline proto/v1/admin_api.proto]: This does not load stores.

File: proto/v1/admin_api.proto [hunk 7]
```diff
@@ -103,28 +103,78 @@ service StoreService {
...
     // the store object back to the caller
-    rpc CreateStore (Store) returns (Store) {
+    rpc CreateStore (CreateStoreRequest) returns (Store) {
+        option (google.api.http) = {
+            post: "/merchants/{merchantId}/stores"
+            body: "*"
+        };
+    }
+
+    // Create store creates the store for the particular merchant and returns
```
Comment (rysavyvladan) [inline proto/v1/admin_api.proto]: Update

File: proto/v1/admin_api.proto [hunk 8]
```diff
@@ -165,7 +215,8 @@ message GetInventoryItemRequest {
     string inventoryId = 2;
 }
 
-message UpdateInventoryItemRequest {
+// Request for creating and updating product items within inventory.
+message AddInventoryItemRequest {
```
Comment (radobar) [inline proto/v1/admin_api.proto]: Isn't this only for adding an item?

**Review by rysavyvladan — CHANGES_REQUESTED** [review]

File: proto/v1/admin_api.proto [hunk 9]
```diff
@@ -177,16 +228,20 @@ message UpdateInventoryItemRequest {
     uint32 count = 51;
 }
 
-message CreateInventoryItemRequest {
+// Request for creating and updating product items within inventory.
```
Comment (radobar) [inline proto/v1/admin_api.proto]: And this only for updating?

**Review by radobar — CHANGES_REQUESTED** [review]

File: proto/v1/admin_api.proto (hunk 4 above)
Comment (rysavyvladan) [inline proto/v1/admin_api.proto]: Update

File: proto/v1/admin_api.proto (hunk 9 above)
Comment (rysavyvladan) [inline proto/v1/admin_api.proto]: Update

**Review by rysavyvladan — APPROVED** [review]

**Pull request merged**